fauxbook.ipynb

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
<p>You've missed out the <code>p</code>  in <code>print</code>.  Getting tired of reviewing this sort of thing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HomelessSandwich/data-science-testing/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>